### PR TITLE
Split the REST API plugin (bsc#1139747) 

### PR DESCRIPTION
**Pull request by lslezak**
- This is related to
  - https://github.com/libyui/libyui/pull/143
  - https://github.com/libyui/libyui-rest-api/pull/2
- Just moving the Qt files to a separate package, adding the packaging and a small cleanup
- The Travis fails because it depends on [this libyui-rest-api change](https://github.com/libyui/libyui-rest-api/pull/2)

### CODE REVIEW

File: PROJECTINFO.cmake [hunk 1]
```diff
@@ -0,0 +1,15 @@
+##### PROJECTINFO for CMake
+
+SET( BASELIB		"yui" )		# don't change this
```
Comment (shundhammer) [inline PROJECTINFO.cmake:3]: Is this still correct now that it's no longer of the big thing that included everything?

Comment (lslezak) [reply PROJECTINFO.cmake:3]: I do not know that Cmake magic, why it's there. But that's the same in all UI plugins, even in the normal Qt ([here](https://github.com/libyui/libyui-qt/blob/master/PROJECTINFO.cmake#L3)) or Ncurses ([here](https://github.com/libyui/libyui-ncurses/blob/master/PROJECTINFO.cmake#L3)). It was just copy&paste...

Comment (shundhammer) [reply PROJECTINFO.cmake:3]: OK then.

**Comment by shundhammer** [comment]
LGTM (but Travis doesn't like it; I guess because the other parts are not there yet)

**Pull request merged**